buildSrc/src/main/java/Dependencies.kt

```diff
@@ -33,7 +33,7 @@ object Versions {
     const val room = "1.1.1"
     const val paging = "1.0.1"
 
-    const val mozilla_appservices = "0.20.2"
+    const val mozilla_appservices = "0.22.0"
```

Comment:
We should land this separately and deal with any potential fallback orthogonality to the bookmarks work.

Reply:
This is complicated by the fact that the arguments and return values for many of the new bookmarks-related classes use types from application-services. I could create new types, but they would mirror much of the existing data classes.

Reply:
Being resolved in #2515